zerver/lib/user_topics.py

```diff
@@ -117,8 +117,9 @@ def set_user_topic_visibility_policy_in_database(
     visibility_policy: int,
     recipient_id: Optional[int] = None,
     last_updated: Optional[datetime.datetime] = None,
-    ignore_duplicate: bool = False,
-) -> None:
+) -> bool:
+    skip_send_event: bool = False
```

Comment:
I think this would be better expressed with having a comment explaining that this function returns True if it modified the database, and False otherwise. The caller interprets that as whether to send an event or not, but I think logically that's what this function is telling you.